> `submariner` would also need an exception for `pkg/apis/vendor`, if we wanted to play it safe. Wouldn’t it be better to provide a way for projects to add their own exceptions?

I sent a second commit with an attempt at this, but I'm struggling with the bash-inside-Makefile peculiarities. Suggestions/feedback appreciated @skitt. :pray: 

I'm also now seemingly blocked by some infra issue (for the last hour or so)

```
[+] Building 30.2s (3/3) FINISHED                                                                                                            
 => [internal] load build definition from Dockerfile.shipyard-linting                                                                   0.1s
 => => transferring dockerfile: 1.70kB                                                                                                  0.0s
 => [internal] load .dockerignore                                                                                                       0.1s
 => => transferring context: 117B                                                                                                       0.0s
 => ERROR [internal] load metadata for docker.io/library/alpine:latest                                                                 30.0s
------
 > [internal] load metadata for docker.io/library/alpine:latest:
------
Dockerfile.shipyard-linting:1
--------------------
   1 | >>> FROM alpine
   2 |     
   3 |     ENV DAPPER_HOST_ARCH=amd64 SHELL=/bin/bash \
--------------------
error: failed to solve: alpine: failed to do request: Head "https://registry-1.docker.io/v2/library/alpine/manifests/latest": dial tcp: i/o timeout
make: *** [Makefile.images:24: package/.image.shipyard-linting] Error 1
```

--- a/Makefile.inc
+++ b/Makefile.inc
@@ -142,7 +142,11 @@ lint: gitlint golangci-lint markdownlint yamllint
 
 # [markdownlint] validates Markdown files in the project
 markdownlint:
-	markdownlint markdownlint -c .markdownlint.yml -i vendor .
+	md_ignored=($(patsubst %/modules.txt,%,$(VENDOR_MODULES))); \
+	if [ -r .mdignore ]; then \
+		md_ignored+=($$(< .mdignore)); \
+	fi; \
+	markdownlint -c .markdownlint.yml $${md_ignored[@]/#/-i } .
 
 # [yamllint] validates YAML files in the project
 yamllint:

